hail_search/test_search.py

```diff
@@ -245,6 +245,29 @@ async def test_frequency_filter(self):
             omit_sample_type='SV_WES',
         )
 
+    async def test_annotations_filter(self):
```

Comment:
It is unclear to me whether the tests cover the cases of allowed transcript filters.

Reply:
It does? Thats what the final 3 assertions in this test are testing. What case are you worried about that you don't see here?

Reply:
My concern was if any variants were filtered out by the transcript consequences with [the code](https://github.com/broadinstitute/seqr/blob/12c49bc8c2a326a76791543fce9a8df473d2a411/hail_search/hail_search_query.py#L663C24-L668). But I think the assertion with `missense` annotations has done the test.